client/my-sites/email/email-non-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { ResponseDomain } from 'calypso/lib/domains/types';
+import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+import './style.scss';
+
+type EmailNonDomainOwnerMessageProps = {
+	domain?: ResponseDomain;
+	selectedSite?: SiteDetails | null;
+	usePromoCard: boolean;
+	source: 'email-comparison' | 'email-management';
+};
+
+export const EmailNonDomainOwnerMessage = ( props: EmailNonDomainOwnerMessageProps ) => {
+	const { domain, selectedSite, source, usePromoCard } = props;
+
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const ownerUserName = useDomainOwnerUserName( selectedSite, domain );
+
+	const isPrivacyAvailable = domain?.privacyAvailable;
+
+	const buildQueryString = ( parameters = {} ) =>
+		parameters ? stringify( parameters, { addQueryPrefix: true, skipNulls: true } ) : '';
+
+	const buildLoginUrl = () => {
+		const redirectUrlParameter = encodeURIComponent(
+			emailManagementPurchaseNewEmailAccount(
+				selectedSite?.slug ?? '',
+				domain?.name ?? '',
+				null,
+				'login-redirect'
+			)
+		);
+
+		return `/log-in/${ buildQueryString( {
+			email_address: ownerUserName,
+			redirect_to: redirectUrlParameter,
+		} ) }`;
+	};
+
+	const contactOwnerUrl = `https://privatewho.is/${ buildQueryString( {
+		s: domain?.name,
+	} ) }`;
+
+	const loginUrl = buildLoginUrl();
+
+	const onClickLink = ( eventType: 'owner_contact' | 'user_login' | 'contact_support' ) => {
+		const properties = {
+			owner_login: ownerUserName,
+		};
+
+		if ( eventType === 'owner_contact' ) {
+			dispatch( recordTracksEvent( `calypso_email_providers_owner_contact_click`, properties ) );
+
+			return;
+		} else if ( eventType === 'contact_support' ) {
+			dispatch( recordTracksEvent( `calypso_email_providers_contact_support_click`, properties ) );
+
+			return;
+		}
+
+		dispatch( recordTracksEvent( `calypso_email_providers_user_login_click`, properties ) );
+	};
+
+	const translateOptions = {
+		components: {
+			loginLink: (
+				<a
+					rel="noopener noreferrer"
+					href={ loginUrl }
+					onClick={ () => onClickLink( 'user_login' ) }
+				/>
+			),
+			reachOutLink: isPrivacyAvailable ? (
+				<a
+					href={ contactOwnerUrl }
+					onClick={ () => onClickLink( 'owner_contact' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			) : (
+				<></>
+			),
+			contactSupportLink: (
+				<a
+					href="https://wordpress.com/help/contact"
+					onClick={ () => onClickLink( 'contact_support' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			),
+		},
+		args: {
+			ownerUserName,
+			selectedDomainName: domain?.name,
+		},
+	};
+
+	function getReason() {
+		switch ( source ) {
+			case 'email-comparison':
+				return translate(
+					'Email service can only be purchased by {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}}, ' +
```

Comment:
```suggestion
					'Email service can only be purchased by %(ownerUserName)s, ' +
```
I think it's hard to know what to expect from `reachOutLink` in this sentence, and given that it has such "dramatic" effects (i.e. links to an unfamiliar external site), I think it's preferable to just skip it altogether here.

Reply:
We should discuss this with @wongasy, this is part of the implementation details and requirements.

Reply:
Discussed in Slack. Removing the link :)